### PR TITLE
Add initial .travis.yml

### PR DESCRIPTION
**Pull request by tianon**

### CODE REVIEW

**Comment by tianon** [comment]
https://travis-ci.org/tianon/gb/builds/90598178 build in progress

**Comment by davecheney** [comment]
LGTM. I realised you have commit, so merge this when you're happy that it passed and we can setup the travis webhooks.

File: vendor/repo_test.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func TestDeduceRemoteRepo(t *testing.T) {
 			url: "https://bitbucket.org/davecheney/hgrepo",
 		},
 		extra: "/cmd/main",
-	}, {
+	}, { */
```
Comment (tianon) [inline vendor/repo_test.go]: I know _you_ won't forget, but just as a public reminder, we do need to figure out what to do about these bitbucket tests. :smile:

**Comment by tianon** [comment]
Everything's green!  You get a CI, you get a CI, everyone gets a CI!

File: vendor/repo_test.go (hunk 1 above)
Comment (davecheney) [inline vendor/repo_test.go]: I'll open an issue so I don't forget.

**Pull request merged**